### ISSUE TITLE
basic ELI5 how-to

### ISSUE THREAD

**Issue by kidbrax** [ArturoDent/find-and-transform#31]
I see a lot of information on the different find/replace commands and transforms, but I am not even sure how to get started. Do I just create a json file and add commands? Then how do I run it if so? I didn't see anything like this in the README/docs. Sorry if I missed it.

**Comment by ArturoDent** [comment]
I mention `keybindings.json` and `settings.json` a few times.  For information on using those, see the vscode documentation.
